modules/install/wizard.php

```diff
@@ -26,7 +26,7 @@
             $row = $db->qry_first("SELECT email FROM %prefix%user WHERE email=%string%", $_POST["email"]);
 
             // If found, update password
-            if ($row['email']) {
+            if ($row !== false) {
```

Comment:
What is wrong here with the `if ($row['email']) {` comparision?

Reply:
Because `qry_first` does not return an array if no row is found and thus this would cause a warning, as it tries to access an array element of an boolean.